annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/FastFieldElement.java

```diff
@@ -0,0 +1,42 @@
+package com.exactpro.epfast.annotation.processing;
+
+import com.exactpro.epfast.annotations.FastField;
+
+import javax.lang.model.element.Element;
+import javax.lang.model.element.ExecutableElement;
+import javax.lang.model.type.TypeMirror;
+
+public class FastFieldElement {
+    private Element setterMethod;
+
+    private String fieldName;
+
+    private TypeMirror parameterType;
+
+    public FastFieldElement(Element setterMethod) {
+        this.setterMethod = setterMethod;
+        this.fieldName = setterMethod.getAnnotation(FastField.class).name();
+        this.parameterType = ((ExecutableElement) setterMethod).getParameters().get(0).asType();
```

Comment:
You can't do it w/o checking for number of parameters first. Move this code to a getter